text_generation/causal_lm/cpp/continuous_batching/library/src/logit_processor.hpp

```diff
@@ -40,7 +44,9 @@ class ILogitTransformer {
     }
 
 protected:
-    double m_value = 0.f;
+    double m_float_value = 0.f;
+    size_t m_uint_value = 0, m_applicable_tensor_len = std::numeric_limits<size_t>::max();
```

Comment:
let's move these fields to derived classes to avoid confusion with _generic_ names